src/components/nodes/pages/summer-course/summer-course-page.tsx

```diff
@@ -0,0 +1,120 @@
+import Wysiwyg from "@components/elements/wysiwyg";
+import {H1, H3} from "@components/elements/headers";
+import {HtmlHTMLAttributes} from "react";
+import Image from "next/image";
+import {NodeSumSummerCourse} from "@lib/gql/__generated__/drupal.d";
+import ArchBanner from "@components/patterns/arch-banner";
+import { convertToLocalDateTime } from "@lib/utils/convert-date";
+
+type Props = HtmlHTMLAttributes<HTMLDivElement> & {
+  node: NodeSumSummerCourse
+  headingLevel?: "h2" | "h3"
+}
+
+const SummerCoursePage = ({ node, ...props }: Props) => {
+  const startDate = node.sumCourseStartDate && convertToLocalDateTime(node.sumCourseStartDate).toUpperCase();
+  const endDate = node.sumCourseEndDate && convertToLocalDateTime(node.sumCourseEndDate).toUpperCase();
+
+  return (
+    <article {...props}>
+      <ArchBanner
+        {...props}
+        aria-labelledby="undefined"
+        imageUrl="/images/temp-bg.jpg"
+        imageAlt=""
+        eagerLoadImage
+      >
+        <div className="w-screen">
+          <div className="flex flex-col justify-center items-center border-archway-dark border-b rs-pb-4 rs-mx-6">
+            {startDate &&
+              <div className="font-sans font-normal rs-mb-0">
+                {startDate}{endDate && (` — ${endDate}`)}
+              </div>
+            }
+            <H1 className="font-normal max-w-[900px] font-roboto text-center rs-mb-0">
+              {node.title}
+            </H1>
+            {node.sumCourseCatalogNumber &&
+              <H3 className="font-normal">{node.sumCourseCatalogNumber}</H3>
+            }
+          </div>
+        </div>
+      </ArchBanner>
+      <div className="centered my-32 grid grid-cols-12 gap-10">
+        <div className="order-2 col-span-12 md:col-span-8">
+          <div className="flex flex-col md:flex-row gap-10 rs-mb-4">
+            {node.sumCourseImage &&
+              <div className="relative aspect-1/1 w-full h-[200px] md:w-[500px] md:h-[500px] *:rounded-[25px]">
+                <Image
+                  className="object-cover"
+                  src={node.sumCourseImage.mediaImage.url}
+                  alt={node.sumCourseImage.mediaImage.alt || ""}
+                  loading="eager"
+                  fill
+                  sizes="100vw"
```

Comment:
We should only use sizes that are appropriate for the area the image is. For this image, you have a max width and height of 200px, then 500px. So the sizes should be `(max-width: 768px) 200px, 500px` (not sure of the 768 though). What this does is it will resize the original image to 200px up to 768 browser width. then it'll resize another for 500px for all other screen sizes.